crates/notion-core/src/session.rs

```diff
@@ -165,26 +165,28 @@ impl Session {
         self.config.get()
     }
 
-    /// Ensures that a platform image has been fully fetched and set up.
-    pub(crate) fn prepare_image(&mut self, platform: &PlatformSpec) -> Fallible<Image> {
+    /// Ensures that a specific Node version has been fetched and unpacked
+    pub(crate) fn ensure_node(&mut self, version: &Version) -> Fallible<()> {
```

Comment:
Aside: I feel like this "prepare"/"ensure" operation needs a better name, but this code is going to be refactored a few more times and it's a purely internal operation so I'm not going to worry about it too much yet.